### ISSUE TITLE
3.8.x, 3.9.x, and 3.10.x releases refer to 3.11 as the latest feature release series 

### ISSUE THREAD

**Issue by deilann** [proevo/pythondotorg#2486]
**Describe the bug**
Release information for various 3.8.x, 3.9.x, and 3.10x versions include the line:

> Note: The release you're looking at is Python 3.9.13, a bugfix release for the legacy 3.9 series. Python 3.11 is now the latest feature release series of Python 3. [Get the latest release of 3.11.x here](https://www.python.org/downloads/).

The issue is merely textual as the link goes to /downloads.

**To Reproduce**
Search for the text "Python 3.11 is now the latest feature release series of Python 3". and the [search results](https://www.python.org/search/?q=Python+3.11+is+now+the+latest+feature+release+series+of+Python+3.) contain all releases with the outdated text.

**Expected behavior**
The text should say 

> Python 3.12 is now the latest feature release series of Python 3. [Get the latest release of 3.12.x here](https://www.python.org/downloads/).

**Comment by hugovk** [comment]
Here's one specific example:

Go to https://www.python.org/downloads/release/python-3913/

It says:

> Note: The release you're looking at is Python 3.9.13, a bugfix release for the legacy 3.9 series. Python 3.11 is now the latest feature release series of Python 3. [Get the latest release of 3.11.x here](https://www.python.org/downloads/).

Indeed, 3.12 is now the latest 3.x.

And it's not just an outdated static page, it says "Release Date: May 17, 2022", and at that time 3.10 was the latest 3.x and at that time it said:

> Note: The release you're looking at is Python 3.9.13, a bugfix release for the legacy 3.9 series. Python 3.10 is now the latest feature release series of Python 3. [Get the latest release of 3.10.x here](https://www.python.org/downloads/).

https://web.archive.org/web/20220517234441/https://www.python.org/downloads/release/python-3913/

I think we need something more sustainable than having to manually edit all patch pages when a new 3.x comes out. Perhaps reword to avoid mentioning the current latest specifically? Maybe something like:

> This release is not the latest feature release series of Python 3.x. [Get the latest Python release here](https://www.python.org/downloads/).

Or we could just leave them as static dated pages, the link takes you to https://www.python.org/downloads/ and you can see the newest versions there.

**Comment by JacobCoffee** [comment]
Who edits these pages? The release manager at the time?

Just chopping off that first line may work for every post future and past I think?
> [Get the latest Python release here](https://www.python.org/downloads/).

side note: I would like to express my profound pain for these (and most other things) not being a codified template instead of whatever they are now for dynamic things like this.

**Comment by hugovk** [comment]
> Who edits these pages? The release manager at the time?

Yes, see 'Create a new “release” for the release' at https://peps.python.org/pep-0101/#how-to-make-a-release
